astropy/utils/iers/iers.py

```diff
@@ -744,37 +734,30 @@ def _refresh_table_as_needed(self, mjd):
             raise ValueError('IERS auto_max_age configuration value must be larger than 10 days')
 
         if (max_input_mjd > predictive_mjd and
-                now_mjd - predictive_mjd > auto_max_age):
+                (now_mjd - predictive_mjd) > auto_max_age):
```

Comment:
This is just easier for me to read without trying to memorize the operator precedence order. It has no impact on performance.